### ISSUE TITLE
wamp.session.on_leave doesn't provide all three positional arguments

### ISSUE THREAD

**Issue by mrrishimeena** [crossbario/crossbar#1691]
As per the documentation in **wamp-proto** [link: [https://wamp-proto.org/_static/gen/wamp_latest.html](https://wamp-proto.org/_static/gen/wamp_latest.html)]

`wamp.session.on_leave` should provide three positional arguments:
1. session|id - The session ID of the session that left
2. authid|string` - The authentication ID of the session that left
3. authrole|string - The authentication role of the session that left

But crossbar doesn't provide **authid** & **authrole**.

**Comment by oberstet** [comment]
yes, the addition of authid and authrole was a recent" change in the WAMP spec ([here](https://wamp-proto.org/_static/gen/wamp_latest_ietf.html#wampsessiononleave)) that hasn't made it into crossbar yet.

the fix is a one line change here:

https://github.com/crossbario/crossbar/blob/2a7ebfd6529645524eb1816efc595590bb58167e/crossbar/router/session.py#L959

**Comment by mrrishimeena** [comment]
When can we expect this change in crossbar docker image?

**Comment by mrrishimeena** [comment]
When can we expect this change in crossbar docker image?

**Comment by mrrishimeena** [comment]
any update related to this issue?

**Comment by mrrishimeena** [comment]
any update related to this issue?
